orca-mine/src/main/groovy/com/netflix/spinnaker/orca/mine/tasks/CompleteCanaryTask.groovy

```diff
@@ -30,8 +30,13 @@ class CompleteCanaryTask implements Task {
     Map canary = stage.context.canary
     if (canary.status?.status == 'CANCELED') {
       new DefaultTaskResult(ExecutionStatus.CANCELED)
+
     } else if (canary.canaryResult?.overallResult == 'SUCCESS') {
       new DefaultTaskResult(ExecutionStatus.SUCCEEDED)
+
+    } else if ( canary.canaryConfig?.continueOnUnhealthy && canary.health.health == "UNHEALTHY") {
+      new DefaultTaskResult(ExecutionStatus.FAILED_CONTINUE)
```

Comment:
Not sure we need the extra whitespace lines ... and I'd add explicit return statements IMO.